pkg/services/supportbundles/supportbundlesimpl/api.go

```diff
@@ -20,16 +21,21 @@ const rootUrl = "/api/support-bundles"
 func (s *Service) registerAPIEndpoints(routeRegister routing.RouteRegister) {
 	authorize := ac.Middleware(s.accessControl)
 
+	orgRoleMiddleware := middleware.ReqGrafanaAdmin
```

Comment:
this would only be the fallback, so I guess we need to grant the fixed role to admin / server admin depending on this flag

Reply:
We also have an inconsistency, as we currently grant the RBAC roles to admin by default. So it will be good to fix that as part of this work. Here's the code where roles are granted:
https://github.com/grafana/grafana/blob/main/pkg/services/supportbundles/supportbundlesimpl/models.go#L38-L48

Reply:
yeah, I noticed the inconsistency :smile:, let's do it properly